js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/graphModals/AddEdgeHandler.tsx

```diff
@@ -0,0 +1,41 @@
+import { Alert, AlertIcon, Button, Stack, Text } from "@chakra-ui/react";
+import React, { useEffect } from "react";
+import { AddEdgeModal } from "./AddEdgeModal";
+import { useGraph } from "../GraphContext";
+
+export const AddEdgeHandler = () => {
+  const { network, selectedAction, setSelectedAction } = useGraph();
+  useEffect(() => {
+    if (network) {
+      if (
+        selectedAction?.action === "add" &&
+        selectedAction.entityType === "edge"
+      ) {
+        network.addEdgeMode();
```

Comment:
Seems like it's not getting disabled. Maybe useEffect is not getting called. Steps 
- Click on Add edge on an empty area. 
- Drag an edge between node
- Cancel the modal
The mode is not yet disabled.

Reply:
added disable on effect cleanup, that was the problem